### PR TITLE
Catalan translations (ca-ES)

### PR DESCRIPTION
**Pull request by socunanena**

### CODE REVIEW

File: src/i18n/ru-RU/results.yml [hunk 1]
```diff
@@ -124,6 +124,8 @@ translations:
 
     - key: options.locale.en-US
       t: Английский
+    - key: options.locale.ca-ES
+      t: Каталонский
```
Comment (sviridoff) [inline src/i18n/ru-RU/results.yml:128]: Каталанский is more commonly used when is referred to the language

**Review by sviridoff — CHANGES_REQUESTED** [review]

**Comment by SachaG** [comment]
Great! Let me know when this is ready to merge :)

**Comment by socunanena** [comment]
> Great! Let me know when this is ready to merge :)

I just fixed a change requested by @sviridoff . For me, it's ready to go to prod if you think all files are ok.

Thanks for your quick response!

**Comment by SachaG** [comment]
:+1:

**Comment by socunanena** [comment]
Thanks!

I'm not able to see it in production, is there anything missing in my code? Or there is just needed some manual action to be deployed?

**Comment by SachaG** [comment]
Oh yeah sorry I will need to redeploy a couple things before it's live. I'll try to do it early next week :)

**Pull request merged**